app/components/TableWithStatusBar/index.tsx

```diff
@@ -0,0 +1,177 @@
+import { useCallback } from 'react';
+import {
+    ActionIcon,
+    Group,
+    Paper,
+    Table,
+    ScrollArea,
+    Collapse,
+    SegmentedControl,
+    Flex,
+    Tabs,
+    SimpleGrid,
+} from '@mantine/core';
+import { useToggle } from '@mantine/hooks';
+import { IoChevronDown, IoChevronUp } from 'react-icons/io5';
+import {
+    MdOutlineTableChart,
+    MdList,
+    MdOutlineGridView,
+    MdOutlineTag,
+    MdOutlineLanguage,
+    MdOutlineCalendarToday,
+    MdOutlineLocationOn,
+} from 'react-icons/md';
+import { _cs } from '@togglecorp/fujs';
+
+import styles from './styles.module.css';
+
+interface Props {
+    className?: string;
+    listColumn?: JSX.Element[];
+    card?: JSX.Element[];
+    listRows?: JSX.Element[];
+    columns?: JSX.Element[];
+    rows: JSX.Element[];
+}
+
+export default function TableWithStatusBar(props: Props) {
+    const {
+        className,
+        listColumn,
+        listRows,
+        columns,
+        rows,
+        card,
+    } = props;
+    const [opened, toggleOpened] = useToggle([true, false]);
+
+    const handleCollapseClick = useCallback(() => {
+        toggleOpened();
```

Comment:
I think you can just use `handleClick={toggleOpened}`. Using an extra callback function does not seem to server any purpose here.

Reply:
Recommend changes gives type error for state action type to mouse event type

Reply:
That's fine for now then.